ui/components/app/currency-input/hooks/useTokenExchangeRate.tsx

```diff
@@ -66,6 +66,8 @@ export default function useTokenExchangeRate(
       contractExchangeRates[tokenAddress] || exchangeRates[tokenAddress];
 
     if (!contractExchangeRate) {
+      // TODO: Fix "Calling setState from useMemo may trigger an infinite loop"
```

Comment:
Do we have some strategy for addressing these kind of things that are out of scope on this PR?